src/lib/tabs/tab-nav-bar/tab-nav-bar.ts

```diff
@@ -10,7 +10,6 @@ import {Directionality} from '@angular/cdk/bidi';
 import {coerceBooleanProperty} from '@angular/cdk/coercion';
 import {Platform} from '@angular/cdk/platform';
 import {auditTime, takeUntil} from '@angular/cdk/rxjs';
-import {ViewportRuler} from '@angular/cdk/scrolling';
```

Comment:
We should also remove the `VIEWPORT_RULER_PROVIDER` in the ripple module actually.

Reply:
Good call. Done.